### PR TITLE
disabled navigation to s+L and removed issuance

### PR DESCRIPTION
**Pull request by jordaniza**

### CODE REVIEW

File: src/pages/PieIndexLanding.svelte [hunk 1]
```diff
@@ -137,6 +136,8 @@
     }),
   );
 
+  $: console.debug({ composition })
```
Comment (gbonumore) [inline src/pages/PieIndexLanding.svelte:139]: is this something should go on production?

Comment (jordaniza) [reply src/pages/PieIndexLanding.svelte:139]: no, thank you good spot

**Comment by AdrianAddesso** [comment]
@jordaniza there're two issues to adjust before merging, pls see below

DeFi++ should be showing in the PieVault section
- https://deploy-preview-335--nightly-pools-piedao.netlify.app/#/pies

UMA KPI options page is broken (loads after I hard refresh it). Once in there I can't navigate anywhere else
- https://deploy-preview-335--nightly-pools-piedao.netlify.app/#/kpi-options-campaign

**Comment by jordaniza** [comment]
> @jordaniza there're two issues to adjust before merging, pls see below
> 
> DeFi++ should be showing in the PieVault section
> 
> * https://deploy-preview-335--nightly-pools-piedao.netlify.app/#/pies
> 
> UMA KPI options page is broken (loads after I hard refresh it). Once in there I can't navigate anywhere else
> 
> * https://deploy-preview-335--nightly-pools-piedao.netlify.app/#/kpi-options-campaign

Let's discuss - neither of those issues are appearing for me (DEFI++ is there and KPI page works fine)

**Pull request merged**